.nix/config.nix

```diff
@@ -70,6 +70,11 @@ with builtins; with (import <nixpkgs> {}).lib;
          #   for a complete list of Coq packages available in Nix
          # * <github_login>:<branch> is such that this will use the branch <branch>
          #   from https://github.com/<github_login>/<repository>
+         hierarchy-builder.override.version = "math-comp:saturate-filter";
```

Comment:
@pi8027 are we waiting for some new release of HB?

Reply:
I wanted to test hierarchy-builder#414 (see the last commit), but this can be done in a separate PR.

Reply:
Ok, indeed let's do that in another PR and move toward merging the current one. It's already way too big.